.circleci/config.yml

```diff
@@ -52,12 +53,12 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - conda-deps-v1-{{ checksum "python/environment.yml" }}
+            - conda-deps-v1-{{ checksum "python/conda_version.txt" }}-{{ checksum "python/environment.yml" }}
```

Comment:
Why do we have a dependency on the conda version now?

Reply:
I thought my issues were due to an outdated conda version, but turns out that was a red herring. I'll remove this.